### ISSUE TITLE
Need help?

### ISSUE THREAD

**Issue by zhan9san** [cloudalchemy/ansible-node-exporter#266]
**What happened?**

@paulfantom @SuperQ 

Sorry to make you involved. I noticed you are the most active maintainer, https://github.com/cloudalchemy/ansible-node-exporter/graphs/contributors

This is really a great project. Thanks to team for your effort. It saves me much more time to deploy `node_exporter` on many servers.

But it seems there has been no update in this repo for several months.

There are some PRs I am looking forward to. e.g.  #180 and  #263 

**Did you expect to see some different?**

I am glad to help review and merge the PR if you don't mind.

**Comment by zhan9san** [comment]
As this repo has been inactive for about 7 months, and no new release for over 1 year, I tried to reach out to the maintainers via email, but no response for 9 days.

I'd like to maintain the forked repo, https://github.com/devops37/ansible-node-exporter

Any issue or pr is welcomed.

Besides, thanks for the maintainers providing us with this great project.

**Comment by SuperQ** [comment]
Sadly me and @paulfantom have not had time to keep maintaining these roles.

We had the idea that we wanted to convert the whole project over to a new-style Ansible role collection, so that the common code toil would be reduced.  Right now a lot of the reason we haven't done much is because the automated code sync from our skeleton project is a bit broken.

**Comment by zhan9san** [comment]
@SuperQ 

Thanks for your reply.

I am glad to hear that you are starting the new project.

It may take some time to have new project ready to use.
There may be no ETA for open source project.

During this time, I am not sure whether you have time to merge existing PRs and create a new release.

Or could you kindly grant me permission?

Appreciate your time.

**Comment by tpoindessous** [comment]
> I'd love to get this idea going again, we talked about starting a new project under `prometheus-community`.

Hi ! And thanks for your work !

Do you have some news to share about this project ? Do you known what is the right way to install node-exporter with ansible ? 

Thanks !

**Comment by SuperQ** [comment]
We have a new repo here: https://github.com/prometheus-community/ansible

**Comment by nlamirault** [comment]
@SuperQ could you expose the new Prometheus Collection into Ansible Galaxy ?

**Comment by SuperQ** [comment]
Yes, it's on the TODO list. We need to find time to add the release automation.

**Comment by antonr-p2p** [comment]
You can find fixed version in https://github.com/prometheus-community/ansible/tree/main/roles/node_exporter
